docs/src/pages/ui/primitives/menu/examples/sizeExample.tsx

```diff
@@ -0,0 +1,20 @@
+import { Menu, MenuItem, Flex } from '@aws-amplify/ui-react';
+
+export const SizeExample = () => {
+  return (
+    <Flex direction="column" width="4rem">
+      <Menu size="small">
+        <MenuItem>Download</MenuItem>
+        <MenuItem>Create a Copy</MenuItem>
+      </Menu>
+      <Menu>
+        <MenuItem>Download</MenuItem>
+        <MenuItem>Create a Copy</MenuItem>
+      </Menu>
+      <Menu size="large">
+        <MenuItem>Download</MenuItem>
+        <MenuItem>Create a Copy</MenuItem>
+      </Menu>
+    </Flex>
+  );
```

Comment:
I noticed that, while the icons for these different menus are different sizes, the actual size of each menu is the same. It seems like Flex stylings will overwrite stylings from the size attribute.

Reply:
Hmm, I was having this problem before, let me make sure it's still fixed.

Reply:
Dropdowns are sizing correctly, and the buttons are scaled based on the icon sizes. I'll create a ticket to address icon size.

Reply:
It looks like the heights are actually different and it's just the widths that are the same since they inherit from Flex.